gatsby-config.js

```diff
@@ -44,14 +55,22 @@ module.exports = {
         path: `${__dirname}/content/blog-posts/`,
       },
     },
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: 'events',
+        path: `${__dirname}/content/events/`,
+      },
+    },
     'gatsby-plugin-image',
     'gatsby-transformer-sharp',
     {
       resolve: 'gatsby-plugin-sharp',
       options: {
         defaults: {
           quality: 85,
-          placeholder: 'none',
+          placeholder: 'dominantColor',
+          transformOptions: { cropFocus: 'attention' },
```

Comment:
I'm not sure if we need the placeholder for the whole website. I would suggest adding the placeholder to the specific image, where you would like it.